src/søknad/steg/4-barnasbosted/BarnasBosted.tsx

```diff
@@ -0,0 +1,215 @@
+  }
+
+  const visHvordanPraktiseresSamværet = (valgtSamværsrett: string) => {
+    return valgtSamværsrett == intl.formatMessage({ id: 'barnasbosted.spm.jaIkkeKonkreteTidspunkt' });
+  }
+
+  const barn = søknad.person.barn[0];
+
+  const bosted = barn.harSammeAdresse ? intl.formatMessage({ id: 'barnekort.adresse.registrert' }) : intl.formatMessage({ id: 'barnekort.adresse.uregistrert' });
+
+  return (
+    <>
+      <Side
+        tittel={intl.formatMessage({ id: 'barnasbosted.sidetittel' })}
+        nestePath={nesteRoute.path}
+        tilbakePath={forrigeRoute.path}
+      >
+        <div className="barnas-bosted">
+          <div className="barnas-bosted__header">
+            <img alt="barn" className="barneikon" src={barn1} />
+          </div>
+          <div className="barnas-bosted__info">
+            <Element className="navn">{barn.navn}</Element>
+            <div className="inforad">
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.fødselsnummer'})}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{barn.fnr}</Normaltekst>
+              </div>
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.alder'})}}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{barn.alder} år</Normaltekst>
+              </div>
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.bosted'})}}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{bosted}</Normaltekst>
+              </div>
+            </div>
+          </div>
+          <div className="barnas-bosted__innhold">
+          <KomponentGruppe>
+            <FeltGruppe>
+            <Element>{barn.navn}{intl.formatMessage({ id: 'barnasbosted.element.andreforelder' })}</Element>
+            </FeltGruppe>
+          <FeltGruppe>
+            <Input className="foreldre-navn-input" onChange={(e) => settForelder({...forelder, "navn": e.target.value})} label="Navn" />
+          </FeltGruppe>
+          </KomponentGruppe>
+          <KomponentGruppe>
+          <div className="fødselsnummer">
+            <div className="fødselsdato">
+          <Normaltekst>{intl.formatMessage({ id: 'datovelger.fødselsdato' })}</Normaltekst>
+          <div className="barn-datovelger">
+          <div className={'datepicker__container'}>
+            <DatePicker
+                    onChange={(e) => settForelder({...forelder, "fødselsdato": e})}
+                    selected={new Date()}
+                    dateFormat={'dd.MM.yyyy'}
+                    className={'datovelger__input'}
+                />
+                </div>
```

Comment:
Her kunne du kanskje brukt `<Datovelger/>` som baserer seg på `DatePicker `:) 
Da sender du inn:
```  
valgtDato: Date | undefined;
tekstid: string;
datobegrensning: DatoBegrensning;
settDato: (date: Date | null) => void;
```

Det datovelger gir oss, er at den setter dato til dagensDato (en konstant) by default når den først rendres. 
Norske dager blir også satt by default ut ifra locale i språkContext (altså, siden norsk er valgt by default, så vil norsk settes her. Vanligvis er default engelsk i React-Datepicker). 

Den kan, seff, når som helst tweakes ytterligere til vårt behov ✨ 

**NB!** Hvis du rendrer flere enn én datovelger samtidig, så vil ikke DagensDato settes riktig ved default siden den siste alltid vil oversskrive staten til de som rendres først. Har ikke funnet en god løsning på dette, men har en midlertidig fiks. Husk å "sjekke" om den forrige datoen er blitt satt. 

I PR-en ["Refaktorering av bosituasjon" ](https://github.com/navikt/familie-ef-soknad/pull/55/files) , i komponent `SøkerSkalFlytteSammenEllerFåSamboer.tsx`, så rendres to datovelgere. 
På kodelinje 96, kan du se at jeg sjekker om dato-feltet for datovelgeren som rendres først, eksisterer (som vil si at den er satt / rendret ferdig) før `<OmSamboerenDin/>` rendres og som inneholder datovelger nr 2. 

Bare huk tak i meg hvis du lurer på noe! :)

Reply:
Nice, byttet til denne :)